integration-cli/docker_cli_cp_to_container_unix_test.go

```diff
@@ -14,6 +14,29 @@ import (
 	"github.com/go-check/check"
 )
 
+func (s *DockerSuite) TestCpToHostWithPermissions(c *check.C) {
+	testRequires(c, SameHostDaemon, DaemonIsLinux)
+
+	tmpDir := getTestDir(c, "test-cp-to-host-with-permissions")
+	defer os.RemoveAll(tmpDir)
+
+	makeTestContentInDir(c, tmpDir)
+
+	containerName := "permtest"
+
+	_, exc := dockerCmd(c, "create", "--name", containerName, "debian", "/bin/bash", "-c", "stat -c '%u %g %a' /permdirtest /permdirtest/permtest")
```

Comment:
`stat` check can be simplified to `stat -c '%u %g %a' /permdirtest/permtest` in conjunction with https://github.com/docker/docker/pull/28923/files#r102597103